components/config/definition.rst

```diff
@@ -395,6 +395,45 @@ with ``append()``::
 This is also useful to help you avoid repeating yourself if you have sections
 of the config that are repeated in different places.
 
+The example results in the following:
+
+.. code-block:: yaml
+
+    database:
+        connection:
+            driver:               ~ # Required
+            host:                 localhost
+            username:             ~
+            password:             ~
+            memory:               false
+            parameters:           # Required
+    
+                # Prototype
+                name:
+                    value:                ~ # Required
+
+.. code-block:: xml
+
+    <config>
```

Comment:
no, we are talking about config usage standalone and not inside a DI extension in this article. so you should just replace the `<config>` element with `<database>`.